currency_rate_update/model/company.py

```diff
@@ -56,5 +57,5 @@ def button_refresh_currency(self):
     # List of services to fetch rates
     services_to_use = fields.One2many(
         'currency.rate.update.service',
-        'company_id',
+        'company_id', #company_id
```

Comment:
I guess, this remained, but is not needed. As I would like the pull request to be accepted in the original OCA repo, we need as few changes as necessary.